style.css

```diff
@@ -63,4 +63,367 @@
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  .about_description {
+    width: 327px;
+    height: 147px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    margin-top: -4.3rem;
+  }
+
+  .contacts {
+    margin-left: -2.5rem;
+    display: flex;
+    list-style: none;
+  }
+
+  .resume {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 162px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .skill_list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 327px;
+    height: 464.15px;
+    margin-top: 10rem;
+  }
+
+  .pgrm_languages {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 327px;
+    height: 320.15px;
+  }
+
+  .lang_head {
+    width: 327px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    margin-top: -6rem;
+  }
+
+  .lang_head_icon {
+    width: 15px;
+    height: 15px;
+    margin-left: 18rem;
+  }
+
+  h6 {
+    margin-left: -19rem;
+    height: 24px;
+    left: 12px;
+    right: 78px;
+    top: calc(50% - 24px / 2);
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+  }
+
+  .language_list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 12px;
+    gap: 12px;
+    width: 327px;
+    height: 248.15px;
+    margin-left: 8rem;
+  }
+
+  .lang_item {
+    display: flex;
+    align-items: center;
+    width: 303px;
+    height: 66.72px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    list-style-type: none;
+    margin: 1rem;
+    padding: 0;
+    margin-left: -3rem;
+  }
+
+  .langs {
+    height: 24px;
+    top: calc(50% - 24px / 2 - 0.36px);
+    font-family: monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 20px;
+    color: #253858;
+    margin-left: 4rem;
+  }
+
+  .langs_img {
+    width: 48px;
+    height: 48px;
+    left: 12px;
+    background: none;
+    border: none;
+    margin-left: -8rem;
+    float: right;
+  }
+
+  #ht {
+    margin-left: -6.5rem;
+  }
+
+  #cs {
+    margin-left: -5.3rem;
+  }
+
+  .frameworks {
+    width: 327px;
+    height: 72px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  .frame_line {
+    height: 1px;
+    background: #dfe1e6;
+  }
+
+  .frame_icon {
+    width: 8px;
+    height: 14px;
+    background: #505f79;
```

Comment:
- [ ] Kindly You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] i would kindly suggest you to work more on this section design not look like the original design on figma please fix the frame work and skill they are not align like the original design on figma template please fix this to make your website look professional. 
- [ ] figma design
![Screenshot 2023-06-03 044532](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/72a80d39-7b05-4045-8b2c-f93721215bde)

- [ ] your design
![Screenshot 2023-06-03 034314](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/07d061b5-c7e9-4e83-9c49-8435596dc5bb)